src/ClusterBootstrap/template/storage/auto_share/setup_lustre.sh

```diff
@@ -40,6 +40,11 @@ touch {{cnf["folder_auto_share"]}}/lustre_setup_finished
 chown -R {{cnf["lustre_user"]}}:{{cnf["lustre_user"]}} {{cnf["folder_auto_share"]}}
 # we further setup service that examine 2 token: when lustre_setup_finished exist, lustre_nodes_locally_mounted doesn't, execute following script
 source ../boot.env
-python ./cloud_init_mkdir_and_cp.py -p file_map.yaml -u $USER -m $MOD_2_CP
+
+# file system operations in Centos cloud init is extremely slow - a few to 
```

Comment:
to confirm, is it cloudinit that make it slow?

Reply:
Correct. If I start a VM and manually execute the command, the latency is good.

Reply:
We should keep this in mind in case there are other roles requiring centos in the future.